Automattic/jetpack
# Tiled gallery: fix 'from' transforms

Fixes #20264

#### Changes proposed in this Pull Request:
* Adds a default columnWidths attributes so saved content is created when transforming from other galleries

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Check out PR to local dev env
* Add a Jetpack slideshow gallery and then transform it to a  Jetpack tiled gallery and make sure the image elements are saved in the post content
* Save the post and reload and make sure images still show in the tiled gallery
* Repeat for the core gallery and a single core image block

Before:
![tiled-before](https://user-images.githubusercontent.com/3629020/128446480-c1b1831d-2a1b-4588-ac4e-e8f85e29c011.gif)

After:
![after-tiled](https://user-images.githubusercontent.com/3629020/128446495-7c94778b-3bd4-4d61-8e17-63a42625835a.gif)
